package-lock.json

```diff
@@ -0,0 +1,13072 @@
+{
```

Comment:
We try to use `yarn` here, so no `package-lock`needed :) this file can be removed

Reply:
haha lol, old habit from me as i'm generally a npm user. Will do :)

Reply:
hehe, know the feel :D